SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -595,31 +653,31 @@ void USpatialSender::SetAclWriteAuthority(const SpatialLoadBalanceEnforcer::AclW
 
 	const WorkerAttributeSet OwningServerWorkerAttributeSet = { WriteWorkerId };
 
-	EntityAcl NewAcl;
+	EntityAcl* NewAcl = StaticComponentView->GetComponentData<EntityAcl>(Request.EntityId);
```

Comment:
What's the guarantee that this component exists already? Is a race possible where it get's removed after a write is queued? Add a null check?

Reply:
This is only called from processed ACL assignment requests from the enforcer which involves checking the EntityACL data exists, if it doesn't no request would be queued, or sent to this method, I can add a `check()` but based on current usage, it's impossible that this happens